static/bidder-info/blasto.yaml

```diff
@@ -1,14 +1,14 @@
-# Contact support@bizzclick.com to connect with Bizzclick exchange.
-# We have the following regional endpoint sub-domains: US East - us-e-node1, EU: n1, APAC: n3
+# Contact support@blasto.ai to connect with Blasto exchange.
+# We have the following regional endpoint sub-domains: US East: t-us, EU: t-eu, APAC: t-apac
 # Use Host macro to resolve the endpoint based on the datacenter.
 # By default, we will use the US East sub-domain.
 # Please deploy this config in each of your datacenters with the appropriate regional subdomain
-endpoint: 'http://{{.Host}}.bizzclick.com/bid?rtb_seat_id={{.SourceId}}&secret_key={{.AccountID}}'
+endpoint: 'http://{{.Host}}.blasto.ai/bid?rtb_seat_id={{.SourceId}}&secret_key={{.AccountID}}'
```

Comment:
Usage of partial dynamic urls is being discouraged and prebid team in future will be working to avoid usage of dynamic subdomain in endpoint url.

Major concerns with such usage are,

security concerns
The security aspect is alleviated by using a fixed top level domain. Due to the potential harm to hosts, we are strict in this requirement. We are working towards fixing the few adapters currently in violation.

connection performance
The connection performance advice is for your benefit. Client specific subdomains prevent Prebid Server from reusing connections across your clients which results in more connections needed for your adapter. The issue gets worse the more successful you are in attracting new clients. Hosts may choose to disable your adapter due to this behavior.

Reply:
I understand, do you have an alternative solution for such cases?

Reply:
pass `host` as query string if you would like to.

Reply:
@gargcreation1992 Do you mean like this `http://blasto.ai/bid?rtb_seat_id={{.SourceId}}&secret_key={{.AccountID}}&host={{.Host}}`?

I don't think we can do this, we have three regions with this subdomains:
us-e-node1 - us east, n1 - eu, n3 - apac. So this is the easiest way to do it, just to make subdomain dynamic

Reply:
@SyntaxNode Got it! Thus by default in blasto.yaml file I can define such endpoint with us east region
```yaml
# We have the following regional endpoint sub-domains:
# US East: us-e-node1
# EU: n1
# APAC: n3
# Please deploy this config in each of your datacenters with the appropriate regional subdomain
endpoint: 'http://us-e-node1.blasto.ai/bid?rtb_seat_id={{.SourceId}}&secret_key={{.AccountID}}'
```

And then pbs.yaml can be modified accordingly to the region? For example: 
```yaml
adapters:
  blasto:
    endpoint: http://n1.blasto.ai/bid?rtb_seat_id={{.SourceId}}&secret_key={{.AccountID}}
```

Reply:
Yes, exactly. You're also welcomed to disable the adapter by default with a placeholder endpoint like in the Rubicon adapter:

```
# Contact global-support@magnite.com to ask about enabling a connection to the Magnite (formerly Rubicon) exchange.
# We have the following regional endpoint domains: exapi-us-east, exapi-us-west, exapi-apac, exapi-eu
# Please deploy this config in each of your datacenters with the appropriate regional subdomain
endpoint: "https://REGION.rubiconproject.com/a/api/exchange"
disabled: true
```

Either are good approaches IMHO.

Reply:
fixed